packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -166,10 +166,12 @@ function getFilteredGlobalObjectShapeDescriptors(endowments: object): PropertyDe
 }
 
 export function getResolvedShapeDescriptors(
-    globalObjectShape: object,
+    globalObjectShape?: object,
```

Comment:
I still don't understand why is this optional?

Reply:
This was not intentional.